includes/reader-activation/class-reader-activation.php

```diff
@@ -1421,5 +1445,23 @@ public static function set_password_reset_mail_content_type() {
 		};
 		add_filter( 'wp_mail_content_type', $email_content_type );
 	}
+
+	/**
+	 * Rate limit password reset.
+	 *
+	 * @param \WP_Error      $errors    A WP_Error object containing any errors generated
+	 *                                  by using invalid credentials.
+	 * @param \WP_User|false $user_data WP_User object if found, false if the user does not exist.
+	 *
+	 * @return \WP_Error
+	 */
+	public static function rate_limit_lost_password( $errors, $user_data ) {
+		if ( $user_data && self::is_reader_email_rate_limited( $user_data ) ) {
+			$errors->add( 'newspack_password_reset_interval', __( 'Please wait before requesting another password reset email.', 'newspack' ) );
```

Comment:
Nit: this could be a bit more specific to help readers understand how long they need to wait.

```suggestion
			$errors->add( 'newspack_password_reset_interval', __( 'Please wait a moment before requesting another password reset email.', 'newspack' ) );
```